spec/system/claims/view_organisations_spec.rb

```diff
@@ -57,10 +57,11 @@ def i_am_redirected_to_the_root_path
   end
 
   def i_am_redirected_to_organisation_index(persona)
-    expected_content = persona.schools.many? ? "Organisations" : "Organisation details"
+    user = persona.becomes(User).becomes(Claims::User)
```

Comment:
This `.becomes().becomes()` feels a bit weird. But I'm expecting this will probably go away or change once we've [set up STI on the User model](https://trello.com/c/IB7IpN2x/148-refactor-user-models-to-use-single-table-inheritance).

Reply:
Yeah, it it isn't pretty. On the placements side, I removed the amiguity and just checked for specific words on the page, which I think is a better way to test the front end more generally. I just got lazy here...